project/app/Http/Controllers/StorageController.php

```diff
@@ -62,4 +62,24 @@ public function filter(Request $request): RedirectResponse | Response
         $type = 'item';
         return $this->filterProcedure($request, $type);
     }
+
+    public function update(Request $request, int $id): RedirectResponse
+    {
+        if (!($this->getRole($this->ensureIsNotNullUser(Auth::user())->role)->storage & 12)) {
```

Comment:
Dobrze, że to sprawdzenie nie zostało zapomniane, ja bym pewnie zapomniał

Reply:
Dzięki